src/components/Card/OnlyCard/OnlyCard.js

```diff
@@ -1,25 +1,31 @@
 import { useState } from "react";
-import { CardContent } from "../CardContent/CardContent";
-import CardImage from "../CardImage/CardImage";
-import Card from "../CardWrapper/CardWrapper";
+import { CardImage } from "../CardImage";
+import { CardWrapper } from "../CardWrapper";
+import { CardContent } from "../CardContent";
 
-const OnlyCard = ({ items }) => {
+export const OnlyCard = ({ items, linkUrl, folderList }) => {
```

Comment:
P3:
props중에 linkUrl, folderList는 CardContent에 전달만 될 뿐이니 아래처럼해도 됩니다.
```suggestion
export const OnlyCard = ({ items, ...props }) => {
```
```javascript
<CardContent
    items={items}
    handlePreventLinkClick={handlePreventLinkClick}
    {...props}
/>
```